temoa/data_io/hybrid_loader.py

```
@@ -469,8 +474,11 @@ def _build_efficiency_dataset(
 
     # --- Core Model Structure ---
     def _load_regional_global_indices(
-        self, data: dict, raw_data: Sequence[tuple], filtered_data: Sequence[tuple]
-    ):
+        self,
+        data: Dict[str, object],
+        raw_data: Sequence[tuple[object, ...]],
```

Ruff (0.14.0)

Unused method argument: `raw_data`

(ARG002)